include/sound/sdca_function.h

```diff
@@ -1110,6 +1113,9 @@ struct sdca_function_data {
 	int num_entities;
 	int num_clusters;
 
+	struct sdca_interrupt interrupt[SDCA_MAX_INTERRUPTS];
+	int interrupt_num;
```

Comment:
I don't think this should live here, let the function driver store these, to be continued below...